### ISSUE TITLE
[13.2.2] RGB 값 normalization

### ISSUE THREAD

**Issue by staycozyboy** [woowacampers/d2l-pytorch#20]
> In addition, we normalize the values of the three RGB (red, green, and blue) color channels. The average of all values of the channel is subtracted from each value and then the result is divided by the standard deviation of all values of the channel to produce the output.
> 13.2.2

RGB 값 normalization과 관련하여 다음 2가지가 궁금하여 질문합니다.
1. RGB 값을 normalize하는 목적이 무엇일까요? == 왜 하는 걸까요?
2. RGB 값이 합쳐져서 결국 픽셀의 색이 정해지는 건데, 이 값을 임의로 normalize하면 색이 바뀌지 않나요? 다시 출력할 때, normalize에 대한 inverse를 취해주는 것인지..?

**Comment by staycozyboy** [comment]
### 스터디
- 다혜님:
    - [레퍼런스](https://wjddyd66.github.io/pytorch/Pytorch-StyleTransfer/)의 Post processing Function에서 똑같이 inverse 해주는 것 같았다.
- 지원님:
    - 꼭 필요한 작업은 아닌 것 같다. normalization한 값들로(예를 들어 0~1인 값들로) 학습을 했으면, 굳이 역연산을 해야할 필요가 있는지 모르겠다. d2l 코드들 쭉 봤을 때도 input단에서는 normalization 하는 부분들을 많이 봤는데 출력단에서 inverse해주는 부분은 못본것 같다.
    - 다혜님의 레퍼런스에서는 visualization(이미지를 띄우기 위해서) post processing function 부분이 있는 것 같다.
- 한결님:
    - [normalization 해야하는 이유](https://goodtogreate.tistory.com/entry/Neural-Network-%EC%A0%81%EC%9A%A9-%EC%A0%84%EC%97%90-Input-data%EB%A5%BC-Normalize-%ED%95%B4%EC%95%BC-%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0)

### 정리
1. 표준화 된 입력을 통해 Gradient Descent 및 Bayesian estimation을 보다 편리하게 수행 할 수 있기 때문.
2. normalize한 순간부터 이는 직접적으로 색을 나타내는 값이 아니므로 바뀌어도 상관없다. 또한, RGB 값을 출력하도록 모델을 설계하기 때문에, inverse등의 과정은 필요 없다.
